daemon/src/model.rs

```diff
@@ -108,3 +109,9 @@ pub struct WalletInfo {
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub struct OracleEventId(pub String);
+
+impl Display for OracleEventId {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        self.0.fmt(f)
+    }
+}
```

Comment:
I usually refer to all 3 symbols (`Display`, `Formatter` and `Result`) via `fmt::`. That results in only one import (`use fmt`) and is unambiguous in what you are referring to.